apkid/rules/dex/packers.yara

```diff
@@ -386,3 +386,50 @@ rule crazy_dog_wrapper : packer
     is_dex
     and 2 of them
 }
+
+rule jsonpacker_dex : packer
```

Comment:
Remove `_dex` in the name, we're already changing a DEX rule.

Reply:
I didnt get detected on my end. Did you verify it?
```sh
$ apkid ~/Downloads/e23f0a124fdaba30c07a3c40011dd99240af081cec4cdfcb990c811126867e59.apk 
[+] APKiD 2.1.2 :: from RedNaga :: rednaga.io
[*] /home/edu/Downloads/e23f0a124fdaba30c07a3c40011dd99240af081cec4cdfcb990c811126867e59.apk!classes.dex
 |-> anti_vm : Build.MANUFACTURER check
 |-> compiler : dexlib 2.x
```

Reply:
strange... it did detect when I checked, but maybe I messed something up after... Let me check again.